python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -483,17 +476,27 @@ def _filter_by_project(self, actions, sw_entities, project):
         filtered = []
 
         for action in actions:
+            # The engine_name property of an engine command is defined by
+            # tk-multi-launchapp, and corresponds to the engine that provided
+            # the information necessary to register the launcher. If the action
```

Comment:
Correct me if I am wrong, but you are testing for the presence of a setting from the launcher app to know if the actions are launchers? This seems brittle. If we're hardcoding this to something known, can't we at least hardcode it to app name insteand, `tk-multi-launchapp`? If we are reluctant of hardcoding an app name, maybe it needs to be a config setting or a framework hook?